flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/exceptionhistory/RootExceptionHistoryEntryTest.java

```diff
@@ -150,25 +164,24 @@ void testFromGlobalFailure() throws ExecutionException, InterruptedException {
                         ExceptionHistoryEntryMatcher.matchesGlobalFailure(
                                 rootCause, rootTimestamp, rootFailureLabels.get()));
 
-        final Predicate<ExceptionHistoryEntry> exception0Predicate =
-                ExceptionHistoryEntryMatcher.matchesFailure(
-                        concurrentException0,
-                        concurrentExceptionTimestamp0,
-                        concurrentlyFailedExecutionVertex0.getTaskNameWithSubtaskIndex(),
-                        concurrentlyFailedExecutionVertex0.getCurrentAssignedResourceLocation());
-        final Predicate<ExceptionHistoryEntry> exception1Predicate =
-                ExceptionHistoryEntryMatcher.matchesFailure(
-                        concurrentException1,
-                        concurrentExceptionTimestamp1,
-                        concurrentlyFailedExecutionVertex1.getTaskNameWithSubtaskIndex(),
-                        concurrentlyFailedExecutionVertex1.getCurrentAssignedResourceLocation());
         assertThat(actualEntry.getConcurrentExceptions())
                 .allMatch(
                         exceptionHistoryEntry ->
                                 exception0Predicate.test(exceptionHistoryEntry)
                                         || exception1Predicate.test(exceptionHistoryEntry));
     }
 
+    private Predicate<ExceptionHistoryEntry> getExceptionHistoryEntryPredicate(
```

Comment:
```suggestion
    private Predicate<ExceptionHistoryEntry> triggerFailureAndCreateEntryMatcher(
```
Based on the old name, I didn't expect the triggering of the failure. We might want to mention that in the method name because it's an essential part of each test scenario that the failure is actually registered through a state update in the `ExecutionGraph`